ledger-core/virtual/handlers/vcallrequest.go

```diff
@@ -53,6 +54,10 @@ func (s *SMVCallRequest) Init(ctx smachine.InitializationContext) smachine.State
 }
 
 func (s *SMVCallRequest) stepExecute(ctx smachine.ExecutionContext) smachine.StateUpdate {
+	if s.pulseSlot.State() != conveyor.Present {
```

Comment:
should we duplicate that check here and [there](https://github.com/insolar/assured-ledger/pull/432/files#diff-b9bd8941b93f6eedb3ea9c2e476f935dR132-R135)?

Maybe we can only check in SMExecute?